js/scenes/Debug.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+} from 'react-native'
+
+import Header from '../components/Header';
+import Compass from '../components/Compass';
+import { defaultTextStyle } from '../../config';
+
+export default class Debug extends Component {
+  constructor(props) {
+    super(props);
+
+    this.navigator = this.props.navigator;
+    this.onBackPress = this.onBackPress.bind(this);
+
+    this.state = {
+      latitude: 0,
+      longitude: 0
+    };
+  }
+
+  onBackPress() {
+    this.navigator.pop();
+  }
+
+  // Watch for position for testing the compass
+  componentWillUnmount() { 
```

Comment:
NIT: Can this be after `componentDidMount` so we see setup then teardown when reading.